operator/config/kit-deployment.yaml

```diff
@@ -124,7 +124,7 @@ spec:
       serviceAccountName: kit
       containers:
       - name: manager
-        image: ko://github.com/awslabs/kit/cmd/controller
+        image: ko://github.com/awslabs/kubernetes-iteration-toolkit/operator/cmd/operator
```

Comment:
I'd name this file controller.yaml or something. Or perhaps config/controller/deployment.yaml, config/controller/rbac/yaml